server/lib/resolvers/wrapResolverWithHooks.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash')
+const {log} = require('../util/logger')
+const request = require('request')
+
+exports.wrapResolverWithHooks = function wrapResolverWithHooks (resolver, resolverMapping) {
+  return wrapResolverWithMappingHooks(resolver, resolverMapping)
+}
+
+function wrapResolverWithMappingHooks (resolverFn, resolverMapping) {
```

Comment:
The function has the same signature as `wrapResolverWithHooks`, why not just put everything in there?

Reply:
👍

Reply:
Because I had other things when I have created it. Going to move since it doesn't make sense anymore!